CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 6.11.0 - 2017-03-01
+- PVAYLADEV-609 / PVAYLADEV-703 / Joint development issue #120: Added a partial index to the messagelog database to speed up retrieval of messages requiring timestamping. This should increase proxy performance in cases where the logrecord table is large.
+- PVAYLADEV-685 / Joint development issue #121: Added a system property to deactivate signer's periodic OCSP-response retrieval on both central server and configuration proxy.
+
```

Comment:
I'm not sure it should be mentioned, as it is a documentation addition that has no effect on the software package. I don't know if we have policy or guideline on this, maybe it should be discussed.